infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationSubscriber.java

```diff
@@ -0,0 +1,56 @@
+package org.corfudb.infrastructure.logreplication.infrastructure;
+
+import lombok.Getter;
+import org.corfudb.infrastructure.logreplication.LogReplicationConfig;
+import java.util.Objects;
+
+/**
+ * This class represents a client/subscriber of Corfu Log Replication.
+ *
+ * A subscriber can be identified by a combination of
+ * 1) Replication Model used by it to send/receive data
+ * 2) Application(Client) generating(Source) or consuming(Sink) the data
+ *
+ * The model and client information is contained in the protobuf schema definition of the table to be replicated and is
+ * obtained from the registry table.
+ */
+public class CorfuReplicationSubscriber {
```

Comment:
Is there a significance to having Corfu as the prefix for all the classes in this package ?

Reply:
No.  I kept it because most other classes use it as a prefix.  But I do see that it is redundant since the class is already in the 'corfu' and logreplication package.  Removed the prefix and changed to 'ReplicationSubscriber'/'ReplicationSession' respectively.